### PR TITLE
11.1

### PR DESCRIPTION
**Pull request by dcrp1234**

### CODE REVIEW

**Comment by dcrp1234** [comment]
From section 11.1, 10 images from exercise solution don't have the image description
From section 11.2, 4 images  from exercise solution don't have the image description

File: ptx/sec_space_coord.ptx [hunk 1]
```diff
@@ -894,7 +983,15 @@
                     <caption/>
                     <!-- START figures/figspace4a_3D.asy -->
                     <image xml:id="img_space4a1_3D">
-                      <description></description>
+                      <shortdescription>
+                        Graph of cylinder in space with formula z = y^2..
```
Comment (sean-fitzpatrick) [inline ptx/sec_space_coord.ptx]: double period

File: ptx/sec_space_coord.ptx [hunk 2]
```diff
@@ -944,7 +1041,18 @@
...
                     <image xml:id="img_space4a2_3D">
-                      <description></description>
+                      <shortdescription>
+                        Graph of parabola in space along with a group of lines on the parabola.
+                      </shortdescription>
+                      <description>
+                        <p>
+                          The <m>x</m>, <m>y</m> and <m>z</m> axis are uncalibrated. The parabola is drawn 
+                          on the <m>y</m> and <m>z</m> axis, <m>z</m> being a function of <m>y</m>. There 
+                          is a group of parallel lines called ruling equidistant from each other that are
```
Comment (sean-fitzpatrick) [inline ptx/sec_space_coord.ptx]: rulings (plural) I would think?

File: ptx/sec_space_coord.ptx [hunk 3]
```diff
@@ -1216,7 +1354,16 @@
...
                       <!-- START figures/figspace4f_3D.asy -->
                     <image xml:id="img_space4b3_3D">
-                      <description></description>
+                      <shortdescription>
+                        Graph of function x= sin (z) drawn in space along with the rules.
+                      </shortdescription>
+                      <description>
+                        <p>
+                          The <m>x</m>, <m>y</m> and <m>z</m> axes are uncalibrated. The sine function 
+                          <m>x=sine(z)</m> described previously is used as the area of cross-section to
```
Comment (sean-fitzpatrick) [inline ptx/sec_space_coord.ptx]: x=\sin(z), not sine(z)

File: ptx/sec_space_coord.ptx [hunk 4]
```diff
@@ -1301,7 +1448,19 @@
           <caption/>
         <!-- START figures/figsurf_rev_intro_3D.asy -->
         <image xml:id="img_surf_rev_introa_3D">
-          <description></description>
+          <shortdescription>
+            Graph showing cross- section for surface of revolution.
+          </shortdescription>
+          <description>
+            <p>
+              The y and a axes are drawn from -2 to 2 and the x axis is drawn from 0 to 4.
```
Comment (sean-fitzpatrick) [inline ptx/sec_space_coord.ptx]: math tags are missing throughout this description

File: ptx/sec_space_coord.ptx [hunk 5]
```diff
@@ -1679,7 +1885,15 @@
               <caption/>
                 <!-- START figures/figsurfrev2b_3D.asy -->
               <image xml:id="img_surfrev2b_3D">
-                <description></description>
+                <shortdescription>
+                  Graph showing surface of revolution of function z= sin(x)  around x axis.
+                </shortdescription>
+                <description>
+                  <p>
+                    The surface of equation <m>z= \sin(\sqrt (x^2 +y^2)</m> is made by revolving
```
Comment (sean-fitzpatrick) [inline ptx/sec_space_coord.ptx]: You will need braces for the square root argument: `\sqrt{x^2+y^2}`. And you are missing a closing parenthesis.

File: ptx/sec_space_coord.ptx [hunk 6]
```diff
@@ -2137,7 +2386,17 @@
...
         <image xml:id="img_quadric_coneb_3D">
-          <description></description>
+          <shortdescription>
+            Graph showing elliptic cone opening along the z axis.
+          </shortdescription>
+          <description>
+            <p>
+              The axes are uncalibrated. Two hollow elliptic cones are drawn with vertices at the origin, 
+              one opening along the positive <m>z</m> axis and the other along the negative <m>z</m> axis. 
+              The graph shows three traces, on the plane y=0, the trace is a straight line passing through
```
Comment (sean-fitzpatrick) [inline ptx/sec_space_coord.ptx]: math tags misisng on two equations

File: ptx/sec_space_coord.ptx [hunk 7]
```diff
@@ -2196,7 +2455,16 @@
...
         <image xml:id="img_quadric_conec_3D">
-          <description></description>
+          <shortdescription>
+            Graph showing elliptic cone opening along the z axis.
+          </shortdescription>
+          <description>
+            <p>
+              The axes are uncalibrated. Two hollow elliptic cones are drawn with vertices at the origin, 
+              one opening along the positive <m>z</m> axis and the other along the negative <m>z</m> axis. 
+              A hyperbola is shown in the y=d plane, the two parts of the hyperbola are traced on the elliptic cone.
```
Comment (sean-fitzpatrick) [inline ptx/sec_space_coord.ptx]: math tags missing

File: ptx/sec_vector_intro.ptx [hunk 8]
```diff
@@ -268,7 +289,17 @@
...
                   <image xml:id="img_vect1a">
-                    <description></description>
+                    <shortdescription>
+                      Graph showing two vectors RS and PP&apos;.
+                    </shortdescription>
+                    <description>
+                      <p>
+                        The <m>x</m> and <m>y</m> axes are drawn from <m>-4</m> to <m>4</m>. Two vectors are drawn. 
+                        The vector <m>PP’</m> is in the first quadrant it is downward facing, it starts from 
+                        <m>P=(3, 2)</m> and ends at <m>P&apos;=(5, 1)</m> . The second vector <m>RS</m> from <m>(-3, -2)</m>
```
Comment (sean-fitzpatrick) [inline ptx/sec_vector_intro.ptx]: You can just do `PP'`

**Comment by sean-fitzpatrick** [comment]
Some changes, mostly small, are requested in the line-by-line comments above.

@dcrp1234 when you say some images in the exercise solutions don't have descriptions, do you mean that you weren't able to add them? Or they were missing but now you've added them?

**Comment by dcrp1234** [comment]
Hi Sean,

So the description tags for the images are present but as we discussed in
the last meeting, only the odd number solutions are present for the
exercises, the images aren’t visible on my end so I couldn’t add their
description.

Best,
Prottasha

**Comment by sean-fitzpatrick** [comment]
Oh, right! let me rebuild a version that has all the solutions easily accessible.

**Comment by sean-fitzpatrick** [comment]
If you want to add descriptions for those other images, I've put up a new version of the book (see the post on Teams)

**Comment by sean-fitzpatrick** [comment]
I think I am still seeing some occurrences of `&appos;` in 11.1. The other requested changes are there.

**Comment by sean-fitzpatrick** [comment]
Everything looks OK now. Just one thing: on line 332 of `sec_vector_intro.ptx` you left behind a semi-colon.

Let me know if you are planning to add anything else to this pull request. If not, I will merge.

**Comment by dcrp1234** [comment]
Hi Sean,

Yes I’m planning to add 11.3 and the exercises from 8.3 after which it will
all be done.

Best,
Prottasha

**Comment by sean-fitzpatrick** [comment]
OK, last commit looks good. Please add a comment here when you've added your last commit, so I know whether or not I can merge the pull request.

**Comment by dcrp1234** [comment]
Hi Sean,

Sorry about the late reply, those were the last changes on my end. It was
great working with you.

Best,
Prottasha

**Comment by sean-fitzpatrick** [comment]
Great, thank you!

**Pull request merged**